openshift/setup-teardown/teardown-all.sh

```diff
@@ -42,7 +42,6 @@ cleanApi() {
     removeFromProject ${API_MINIO_BUILD_NAME} ${TOOLS_PROJECT}
     removeFromProject ${API_NODEJS_BUILD_NAME} ${TOOLS_PROJECT}
     removeFromProject ${API_NODEJS_BUILD_NAME}-generator ${TOOLS_PROJECT}
-    removeFromProject ${API_MINIO_BUILD_NAME} ${TOOLS_PROJECT}
```

Comment:
Just confirming we don't want to tear down the Minio deployment because we're using the same one?

Reply:
I'm just removing the duplicate line.  It doesn't need to be called twice

Reply:
Oh I see it now, thank you!